atd-vzv/src/views/summary/CrashesByMonth.js

```diff
@@ -160,14 +160,29 @@ const CrashesByMonth = () => {
 
   return (
     <Container>
-      <Row className="pb-3">
+      <Row>
         <Col>
-          <h3 className="text-center">{crashType.textString} by Year</h3>
+          <h1 className="text-left, font-weight-bold">By Year</h1>
         </Col>
       </Row>
-      <Row style={{ paddingBottom: 20 }}>
+      <Row>
+        <Col>
+          <CrashTypeSelector setCrashType={setCrashType} />
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          <hr />
+        </Col>
+      </Row>
+      <Row>
         <Col>{!!chartData && renderHeader()}</Col>
       </Row>
+      <Row>
+        <Col>
+          <hr style={{ marginTop: 3 }} />
```

Comment:
might be better to use bootstrap spacing classes (ex: `mt-5`) instead of inline styling for theme consistency.

https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
Thanks for catching that!